tutorials/raw/plot_events.py

```diff
@@ -0,0 +1,153 @@
+# using :func:`mne.pick_events`, which has parameters ``include`` and
+# ``exclude``. For example, in the sample data Event ID 32 corresponds to a
+# subject button press, which could be excluded as:
+
+events_no_button = mne.pick_events(events, exclude=32)
+
+###############################################################################
+# It is also possible to combine two Event IDs using :func:`mne.merge_events`;
+# the following example will combine Event IDs 1, 2 and 3 into a single event
+# labelled ``1``:
+
+merged_events = mne.merge_events(events, [1, 2, 3], 1)
+print(np.unique(merged_events[:, -1]))
+
+###############################################################################
+# Note, however, that merging events is not necessary if you simply want to
+# pool trial types for analysis; the next section describes how MNE-Python uses
+# *event dictionaries* to map integer Event IDs to more descriptive label
+# strings.
+#
+#
+# Mapping Event IDs to trial descriptors
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# So far in this tutorial we've only been dealing with integer Event IDs, which
+# were assigned based on DC voltage pulse magnitude (which is ultimately
+# determined by the experimenter's choices about what signals to send to the
+# STIM channels). Keeping track of which Event ID corresponds to which
+# experimental condition can be cumbersome, and it is often desirable to pool
+# experimental conditions during analysis. You may recall that the mapping of
+# integer Event IDs to meaningful descriptions for the :ref:`sample dataset
+# <sample-dataset>` is given in :ref:`this table
+# <sample-data-event-dict-table>` in the :doc:`introductory tutorial
+# <../intro/plot_introduction>`. Here we simply reproduce that mapping as an
+# *event dictionary*:
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
+
+###############################################################################
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data, and the resulting :class:`~mne.Epochs` object allows pooling
+# by requesting partial trial descriptors. For example, if we wanted to pool
+# all auditory trials, instead of merging Event IDs 1 and 2 using the
+# :func:`~mne.merge_events` function, we can make use of the fact that the keys
+# of ``event_dict`` contain multiple trial descriptors separated by ``/``
+# characters: requesting ``'auditory'`` trials will select all epochs with
+# Event IDs 1 and 2; requesting ``'left'`` trials will select all epochs with
+# Event IDs 1 and 3. An example of this is shown in a later tutorial.
+#
+# .. TODO replace above sentence when the relevant tut is ready:
+#    An example of this is shown later, in the :ref:`epoch-pooling` section of
+#    the :ref:`epochs-intro-tutorial` tutorial.
+#
+#
+# Plotting events
+# ^^^^^^^^^^^^^^^
```

Comment:
I find weird that in mne I can always do `foo.plot()` but not for `events`. I'm not sure if we should push for that (maybe not).

Reply:
events are plain arrays on purpose. So people can easily change them